src/store/utils.js

```diff
@@ -33,4 +34,30 @@ export async function stacRequest(cx, link) {
     opts = link;
   }
   return await axios(opts);
-}
\ No newline at end of file
+}
+
+export const convertCoordinatesToEpsg4326 = (stacObj) => {
+    if (stacObj && !!stacObj['crs']) {
+      const stacEpsg = stacObj.crs.init.toUpperCase();
+
+      if (!!stacObj['geometry'] && stacObj['type'] === 'Feature') {
+        const geoCoordinates = stacObj.geometry.coordinates[0];
+        const coordinatesConverted = geoCoordinates.map((elem) => {
+          return proj4(`${stacEpsg}`, "EPSG:4326", [elem[0], elem[1]]);
+        });
+        
+        delete stacObj.crs;
+        stacObj.geometry.coordinates[0] = coordinatesConverted;
+      
+      } else if (!!stacObj['extent'] && stacObj['type'] === 'Collection') {
+        const bboxdCoordinates = stacObj.extent.spatial.bbox[0];
+        const coordinatesConverted = [...proj4(`${stacEpsg}`, "EPSG:4326", [bboxdCoordinates[0], bboxdCoordinates[1]]), ...proj4(`${stacEpsg}`, "EPSG:4326", [bboxdCoordinates[2], bboxdCoordinates[3]])];
+       
+        delete stacObj.crs;
+        stacObj.extent.spatial.bbox[0] = coordinatesConverted;
```

Comment:
Are you sure we need the `[0]` index here? I thought a bbox is a simple array of length 4...

Reply:
By stact spec org the bbox is an Array of Array of numbers. (https://stacspec.org/STAC-ext-api.html#operation/describeCollection)

Reply:
Ye, sure, well caught. Not only that, but prepare the re-ptojection function to deal with multiple level arrays not only for `bbox` but also for `coordinates`. :+1: